src/palace/manager/celery/tasks/reaper.py

```diff
@@ -228,8 +228,7 @@ def loan_reaper(task: Task) -> None:
     now = utc_now()
     deletion_query = delete(Loan).where(
         Loan.license_pool_id == LicensePool.id,
-        LicensePool.unlimited_access == False,  # type: ignore[comparison-overlap]
-        LicensePool.open_access == False,
+        LicensePool.metered_or_equivalent_type == true(),
```

Comment:
Loan reaping is now scoped to `metered_or_equivalent_type` pools. This is more direct then the old comparison, and makes sure we are only targeting expired or outdated metered loans.